app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,10 @@
+class QuotesController < ApplicationController
+
+def show_quote
```

Comment:
We want all action controller's to be uniform, this would be better as a "def show"